internal/app/parcel/repository_test.go

```diff
@@ -67,7 +68,83 @@ func TestRepository_InsertParcel(t *testing.T) {
 
 		repo := NewRepository(sqlxDB)
 		err := repo.InsertParcel(context.Background(), parcel)
-		assert.EqualError(t, err, "sql-error")
+		assert.NotNil(t, err)
+	})
+}
+
+func TestRepository_GetParcelsList(t *testing.T) {
+	parcels := []model.Parcel{
+		{
+			ID:                 0,
+			UserID:             1,
+			CarrierID:          0,
+			Status:             1,
+			SourceAddress:      "Dhaka Bangladesh",
+			DestinationAddress: "Pabna Shadar",
+			ParcelType:         "Document",
+			Price:              200,
+			CarrierFee:         180,
+			CompanyFee:         20,
+		}, {
+			ID:                 0,
+			UserID:             2,
+			CarrierID:          0,
+			Status:             1,
+			SourceAddress:      "Dhaka Bangladesh",
+			DestinationAddress: "Pabna Shadar",
+			ParcelType:         "Document",
+			Price:              200,
+			CarrierFee:         180,
+			CompanyFee:         20,
+		}}
+
+	t.Run("should return success", func(t *testing.T) {
+		db, m, _ := sqlmock.New()
+		defer db.Close()
+
+		sqlxDB := sqlx.NewDb(db, "sqlmock")
+
+		m.ExpectQuery(regexp.QuoteMeta("SELECT * FROM parcel WHERE status=$1 LIMIT $2 OFFSET $3")).
+			WillReturnRows(sqlmock.NewRows([]string{"user_id", "status", "source_address", "destination_address", "type", "price", "carrier_fee", "company_fee"}).
+				AddRow(parcels[0].UserID, parcels[0].Status, parcels[0].SourceAddress, parcels[0].DestinationAddress, parcels[0].ParcelType, parcels[0].Price, parcels[0].CarrierFee, parcels[0].CompanyFee).
+				AddRow(parcels[1].UserID, parcels[1].Status, parcels[1].SourceAddress, parcels[1].DestinationAddress, parcels[1].ParcelType, parcels[1].Price, parcels[1].CarrierFee, parcels[1].CompanyFee))
+
+		repo := NewRepository(sqlxDB)
+		result, err := repo.GetParcelsList(context.Background(), 1, 2, 0)
```

Comment:
It will be good for me to understand var name instead of value. In line 113 => 1, 2, 0 replace it with var name.

Reply:
Please provide a proper example for all the test cases, as I can't use same variable initialization for all cases.